integrationTests/src/test/java/org/sonarqube/gradle/AndroidTest.java

```diff
@@ -315,11 +335,6 @@ public void noDebugVariant() throws Exception {
         baseDir.resolve("src/main/AndroidManifest.xml"));
     assertThat(Paths.get(props.getProperty("sonar.tests"))).isEqualTo(baseDir.resolve("src/test/java"));
 
-    assertThat(Paths.get(props.getProperty("sonar.java.binaries")))
-      .isEqualTo(baseDir.resolve("build/intermediates/javac/release/classes"));
-    assertThat(Paths.get(props.getProperty("sonar.java.test.binaries")))
-      .isEqualTo(baseDir.resolve("build/intermediates/javac/releaseUnitTest/classes"));
```

Comment:
Let's try and see if we can add compilation so that we can keep these assertions